test/Microsoft.TestPlatform.AcceptanceTests/TranslationLayerTests/DiscoverTests.cs

```diff
@@ -167,7 +166,7 @@ public void DiscoverTestsUsingSourceNavigation(RunnerInfo runnerInfo)
             }
             else
             {
-                Assert.AreEqual(22, testCase.FirstOrDefault().LineNumber);
+                Assert.AreEqual(24, testCase.FirstOrDefault().LineNumber);
```

Comment:
Why? Did the file change?

Reply:
See comment above (https://github.com/microsoft/vstest/pull/3268/files#r788654002). I forgot to change the part for the release (will push the fix soon).